report/Evaluation/Essence.tex

```diff
@@ -27,26 +27,29 @@ \subsection{Leverage}
 Some categories were easy to identify, while others were difficult to separate.
 One solution to this is to be more specific when talking about the different categories, both in External and Internal and have more in-depth examples, which can help the reader to understand the terms better.
 
+The reason why we say this, is that there were some assets, that we could classify both in Leverage and Ecology.
+An example of this could be the GitHub API.
+We have not made the GitHub API, which is why it should be an Ecology Interface, as it is trivial to create the interface, as we should just hit a certain endpoint, however, we could argue that the GitHub API, could also be in the Leverage, which means that we should make a crafted interface, so we can interact with the API.
+There is some setup in the API, which means we need to write some code before we can use it, which can be argued that we then have made a crafted interface for the GitHub API.
+Because of this, it was sometimes hard to classify it, because where should the GitHub API be at? 
+Leverage or Ecology?
+So we need a clear line in, what is a "crafted interface" and what is not.
+We know that this is one extreme, but one that should be argued about and found a solution to.  
+
 In regards to the evaluation of the different Leverage Points, we opted to use the PCRT method, to rank and describe each point.
 The method itself was easy to use and work with, however, one problem we had, was that it was hard to distinguish all points, in terms of, if it is about the technology in regards to the project, or if it is only about the technology.
 We might have ranked the Leverage Points differently if we only looked at the technology itself and not looked at it in the context of the project.
 Therefore it could be helpful if there was a clear recommendation on how to adjust for project context or limitations.
 
 Just as in Ecology, we propose that before starting to classify the potential Leverage points, then create a list of these Leverage points, and then classify them.
 
+After some discussions, we think that it does not matter a lot where a given asset is classified in.
+If it is in the Ecology or Leverage, it does not matter that much, unless you need to put in a lot of work, to get the interface to work, then it might be preferred to use the asset as a Leverage Point, then an Ecology Interface.
```

Comment:
```suggestion
If it is in Ecology or Leverage, it does not matter that much, unless you need to put in a lot of work, to get the interface to work, then it might be preferred to use the asset as a Leverage Point, then an Ecology Interface.
```